src/cross_staff_offset.lua

```diff
@@ -4,154 +4,132 @@ function plugindef()
     finaleplugin.Copyright = "CC0 https://creativecommons.org/publicdomain/zero/1.0/"
     finaleplugin.Author = "Carl Vine"
     finaleplugin.AuthorURL = "http://carlvine.com/lua/"
-    finaleplugin.Version = "v1.29"
-    finaleplugin.Date = "2022/07/11"
+    finaleplugin.Version = "v1.33"
+    finaleplugin.Date = "2022/08/05"
 	finaleplugin.MinJWLuaVersion = 0.62
-    finaleplugin.AdditionalMenuOptions = [[  CrossStaff Offset No Dialog  ]]
-    finaleplugin.AdditionalUndoText = [[     CrossStaff Offset No Dialog  ]]
-    finaleplugin.AdditionalPrefixes = [[     no_user_dialog = true  ]]
-    finaleplugin.AdditionalDescriptions = [[ Offset horizontal position of cross-staff note entries - NO DIALOG ]]
```

Comment:
Looks like as is this PR is removing the additional menu options?

Reply:
> Looks like as is this PR is removing the additional menu options?

Yes it is - following on from various discussions that it was more consistent to offer **No Dialog** behaviour via modifier key action that's exactly what this script introduces. (As well as retaining dialog position with RGP 0.64+)